### PR TITLE
Rek add db to index

### PR DESCRIPTION
**Pull request by kilbergr**
This PR creates the endpoint for the index page to display all the issues currently in the db. It includes a test to test same.

It also includes language about running tests in the db and a make command to run both server and client tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,7 @@ server/bin/golint:
 golint: server/bin/golint
 
 deps: golint pre-commit client_deps server_deps
+all_test: client_test server_test
```
Comment (macrael) [inline Makefile]: we could just call it 'test'

File: README.md [hunk 2]
```diff
@@ -46,6 +47,14 @@ The above will start the server running and starts the webpack dev server, proxi
 
 Dependencies are managed by yarn
 
+### Testing
+
+There are a few handy targets in the Makefile to help you run tests:
+
+* `make client_test`: Run frontend testing suites.
+* `make server_test`: Run backend testing suites.
+* `make all_test`: Run both client- and server-side testing suites.
```
Comment (macrael) [inline README.md]: if you decide to change it to just test, change it here too.

File: server/src/dp3/pkg/api/api_test.go [hunk 3]
```diff
@@ -43,17 +43,42 @@ func TestSubmitIssueHandler(t *testing.T) {
...
 
 	resp := w.Result()
 
+	// Then: Expect a 200 status code
 	if resp.StatusCode != 200 {
 		t.Errorf("Returned status code: %d", resp.StatusCode)
 	}
+	// And: Returned query to include our posted issue
+	var m map[string]interface{}
+	json.Unmarshal(rr.Body.Bytes(), &m)
```
Comment (macrael) [inline server/src/dp3/pkg/api/api_test.go]: This is checking on the body of the response to the POST request, I think you should check instead on the body of the GET request.

Comment (kilbergr) [reply server/src/dp3/pkg/api/api_test.go]: D'oh! Thanks :)

**Pull request merged**